lib/world_cup.rb

```diff
@@ -0,0 +1,19 @@
+require './lib/team'
+require './lib/player'
+require 'pry'
+
+class WorldCup
+
+  attr_reader :year, :teams
+  def initialize(year, teams)
+    @year = year
+    @teams = teams
+  end
+
+  def active_players_by_position(position)
+    @teams.flatten.group_by do |x|
```

Comment:
Think that `@teams` is already flattened. It's a array of Team objects. You could do this if you had an array of arrays, but you can't skip over the Team barrier to access their `@players` arrays.